src/main/java/roomescape/domain/ReservationTimeRepository.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.domain;
+
+import java.sql.PreparedStatement;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ReservationTimeRepository {
+    private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+
+    private JdbcTemplate jdbcTemplate;
+
+    public ReservationTimeRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<ReservationTime> findAll() {
+        return jdbcTemplate.query("SELECT id, start_at FROM reservation_time", reservationTimeRowMapper());
+    }
+
+    public Optional<ReservationTime> findById(Long id) {
+        ReservationTime reservationTime = jdbcTemplate.queryForObject(
+                "SELECT id, start_at FROM reservation_time WHERE id = ?",
+                reservationTimeRowMapper(), id);
+        return Optional.ofNullable(reservationTime);
+    }
+
+    public ReservationTime create(ReservationTime reservationTime) {
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement ps = connection.prepareStatement(
+                    "INSERT INTO reservation_time (start_at) VALUES (?)", new String[]{"id"});
```

Comment:
```suggestion
            PreparedStatement ps = connection.prepareStatement(
                    "INSERT INTO reservation_time (start_at) VALUES (?)", 
                    new String[]{"id"}
            );
```
줄바꿈하는 김에 이렇게 해보는건 어떨까요?